lib/upipe-freetype/upipe_freetype.c

```diff
@@ -489,6 +579,94 @@ static void upipe_freetype_input(struct upipe *upipe, struct uref *uref,
     }
 }
 
+/** @internal @This parses an RGBA color.
+ *
+ * @param upipe description structure of the pipe
+ * @param value RGBA value to parse
+ * @param rgba array filled with the r, g, b and a value
+ * @return an error code
+ */
+static int upipe_freetype_parse_rgba(struct upipe *upipe,
```

Comment:
Perhaps this too should be in ubuf_pic ?

Reply:
Fixed